WIP: Add support for PHP 8

internalsystemerror opened the pull request:
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | yes

### Description

This PR adds support for PHP 8.0. Closes #21.

This includes commits from #28.

Commit:
update laminas/laminas-coding-standard to ~2.1.0

Signed-off-by: Gary Lockett <gary@creativecow.uk>

weierophinney commented:
I'm going to revert the phpcs additions - this package actually has no PHP in it, as it's for the UI layer (which is all done in angular 1).

weierophinney committed:
qa: revert addition of phpcs

The only PHP file in here is the `Module` class, which is used to provide asset configuration only; there's no need to do any checks.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>
